llpc/test/shaderdb/relocatable_shaders/PipelineVsFs_ImmutableSampler.pipe

```diff
@@ -9,6 +9,23 @@
 ; SHADERTEST: {{^=====}} AMDLLPC SUCCESS
 ; END_SHADERTEST
 
+// Test that the immutable sampler (descriptorRangeValue) does not affect the
+// hash for the relocatable shaders.
+
+; BEGIN_HASHTEST
+; RUN: mkdir %t && grep -v "descriptorRangeValue" %s > %t/temp.pipe
```

Comment:
I'm not sure of the best way to get a temporary file with the extension that I want.  I can't use `%t` because it does not end with `.pipe`, which means the `amdllpc` command will not interpret it as a pipe file.

Reply:
I think you can just write to `%t.pipe`.
https://wiki.dlang.org/LDC_Lit-based_testsuite#A_simple_test_case_example

Reply:
cool thanks.